File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/MoveIntakeSomeBit.java
package org.firstinspires.ftc.teamcode.commands.low_level;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.IntakeManager;

public class MoveIntakeSomeBit extends InstantCommand {
    public MoveIntakeSomeBit(IntakeManager manager, int i) {
        super(() -> manager.moveSlide(i));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/intake/MoveIntakeSomeBit.kt
package org.firstinspires.ftc.teamcode.commands.low_level.intake

import com.arcrobotics.ftclib.command.InstantCommand
import org.firstinspires.ftc.teamcode.managers.IntakeManager

class MoveIntakeSomeBit(manager: IntakeManager, i: Int) : InstantCommand({ manager.moveSlide(i) })